app/schedule/route.js

```diff
@@ -0,0 +1,39 @@
+import Route from 'ember-route';
+import moment from 'moment';
+
+export default Route.extend({
+  titleToken() {
+    const fullDayAndDate = this.controllerFor('schedule').get('fullDayAndDate');
+    return `Schedule for ${fullDayAndDate}`;
+  },
+  queryParams: {
+    scheduleStation: {
+      refreshModel: true
+    }
+  },
+  model({year, month, day, scheduleStation}) {
+    const apiQueryParams = {};
+
+    // Pulled from query string attached to URL; if nothing, wnyc-fm939
+    apiQueryParams.scheduleStation = scheduleStation;
+
+    // Set API query string scheduleDate to URL's date; otherwise, set to today
+    const scheduleDate = moment(`${year} ${month} ${day}`, 'YYYY MMM DD').format('YYYY-MM-DD');
+    apiQueryParams.scheduleDate = scheduleDate;
+
+    return this.store.query('schedule', apiQueryParams);
+  },
+  setupController: function(controller, model) {
+    controller.setProperties(this.paramsFor('schedule'));
```

Comment:
you don't need all these parameters. you should return the `scheduleDate` from the route along with the schedule query, and then use your template helper to manipulate output strings based on that `scheduleDate` value.